AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/Support/MockModelReconciliatinQueue.swift

```diff
@@ -24,6 +24,7 @@ class MockModelReconciliationQueue: ModelReconciliationQueue {
 
     init(modelSchema: ModelSchema,
          storageAdapter: StorageEngineAdapter?,
+         configuration: DataStoreConfiguration,
```

Comment:
nit: Turns out this file name is misspelled :)

Reply:
yea, i saw that, decided to leave it, but i can update it.